themes/custom/droopler_theme/scss/barrio/_node.scss

```diff
@@ -91,10 +91,16 @@ $node-thumbnail-margin-bottom: 15px !default;
 }
 
 /* Unpublished node styles. */
-.node--unpublished {
-  padding: 20px 15px 0;
-}
 .node--unpublished .comment-text .comment-arrow {
   border-left: 1px solid #fff4f4;
   border-right: 1px solid #fff4f4;
 }
+.unpublished-message {
+  text-align: center;
+  background: red;
```

Comment:
we should use variables
check _colors.scss and find your desired color variable :)